dev-java/bcprov/bcprov-1.65.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
Committing a new ebuild with EAPI 5 today is very... controversial.

Reply:
I'd assumed the eclasses needed it, and that was wrong!